src/fatx_partition.c

```diff
@@ -50,15 +51,30 @@ int fatx_check_partition_signature(struct fatx_fs *fs)
     return FATX_STATUS_SUCCESS;
 }
 
+/*
+ * Initialize the partition with a new superblock.
+ */
+int fatx_init_superblock(struct fatx_fs *fs, size_t sectors_per_cluster)
+{
+    struct timeval time;
+
+    gettimeofday(&time, NULL);
+    fs->volume_id = time.tv_usec;
```

Comment:
Technically the volume ID is just random, but this is what the xbox itself does (effectively) so *shrug*.